generated/nidmm/enums.py

```diff
@@ -139,6 +139,14 @@ class DCNoiseRejection(Enum):
     '''
 
 
+class DigitsResolution(Enum):
+    THREEPOINTFIVE = 3.5
```

Comment:
These names were made up by me. .NET does not appear to use the enum I created them in this fashion because the generator will fail with enums that only contain numbers. Without changing the names, the generator creates an enum with _3 and _4, etc. This causes the build to fail. Is there a way to remove an enum from an attribute in attribute_addon?

We cannot remove this specific enum as it is needed by the Resolution attribute. The resolution attribute was removed on accident and needs to be added back.

Reply:
We should remove the enum and just use numbers.